graders/c/Dockerfile

```
@@ -12,7 +12,7 @@ ENV PYTHONPATH=/cgrader/:/grade/serverFilesCourse
 ENV C_INCLUDE_PATH=/cgrader
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends python3.12 gcc g++ gdb make python3-pip valgrind check pkg-config clang \
+    && apt-get install -y --no-install-recommends python3.12 gcc g++ gdb make python3-pip valgrind check pkg-config clang python3-clang \
```

Nit, keeping Python things adjacent to each other:

```suggestion
    && apt-get install -y --no-install-recommends gcc g++ gdb make valgrind check pkg-config clang python3.12 python3-pip python3-clang \
```

It might be nice to leave a comment calling out why we're installing the `clang` package here instead of via `requirements.txt`, wince that's a bit non-standard.

---

Done in 492080d6717bd0a8a2d8d775ce34446ebc284ee9.